Updating_plugins_for_large_tests.md

```diff
@@ -0,0 +1,33 @@
+#How to run Pluginsync for large tests
+The following steps are necessary for updating plugins to the new large test framework. 
+
+###In the snap-pluginsync repo:
+- Fork and clone repo of plugin you want to run large tests on
+- Fork repo and clone snap-pluginsync
+- Run `$ ./pluginsync.sh`
+  - If this errors make sure ssh keys are set up
```

Comment:
maybe add this step before running pluginsync:

* make sure ssh-agent is running and github ssh key is load by running `ssh-add -l` (it should list your key).